lib/javascripts/base_app.js

```diff
@@ -42,6 +43,11 @@ function registerHelpers(app) {
       return e.message;
     }
   });
+
+  Handlebars.registerHelper('spinner', function(key) {
+    if (_.isObject(key)) { key = "dotted"; }
```

Comment:
I don't get this, why isn't it `if (key === 'dotted')` ? Is it because we don't put quotes around it in the template?